src/components/05_pages/Content/Content.js

```diff
@@ -0,0 +1,218 @@
+          placeholder="Title"
+          onChange={e => {
+            this.setState({ title: e.target.value }, () => {
+              this.props.requestContent(this.state);
+            });
+          }}
+          margin="normal"
+        />
+
+        <FormControl className={styles.formControl}>
+          <InputLabel htmlFor="select-multiple-checkbox">
+            Content Type
+          </InputLabel>
+          <Select
+            multiple
+            value={this.state.contentTypes}
+            onChange={e => {
+              this.setState({ contentTypes: e.target.value }, () => {
+                this.props.requestContent(this.state);
+              });
+            }}
+            input={<Input id="select-multiple-checkbox" />}
+            renderValue={selected => (
+              <div className={styles.chips}>
+                {selected.map(value => (
+                  <Chip
+                    key={value}
+                    label={this.props.contentTypes[value].name}
+                    className={styles.chip}
+                  />
+                ))}
+              </div>
+            )}
+          >
+            {Object.keys(this.props.contentTypes).map(type => (
+              <MenuItem key={type} value={type}>
+                <Checkbox
+                  checked={this.state.contentTypes.indexOf(type) > -1}
+                />
+                <ListItemText primary={this.props.contentTypes[type].name} />
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+
+        <FormControl className={styles.formControl}>
+          <InputLabel htmlFor="status">Status</InputLabel>
+          <Select
+            value={this.state.status || ''}
+            onChange={e => {
+              this.setState({ status: e.target.value }, () => {
+                this.props.requestContent(this.state);
+              });
+            }}
+            input={<Input name="status" id="status" />}
+            autoWidth
+          >
+            <MenuItem value="">
+              <em>Any</em>
+            </MenuItem>
+            <MenuItem value="published">Published</MenuItem>
+            <MenuItem value="unpublished">Unpublished</MenuItem>
+          </Select>
+        </FormControl>
+      </div>
+      <Paper>
+        <Table>
+          <TableHead>
+            <TableRow>
+              {[
+                'Title',
+                'Content Type',
+                'Author',
+                'Status',
+                'Updated',
+                'Actions',
+              ].map(item => <TableCell key={item}>{item}</TableCell>)}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {this.props.nodes.map(
+              ({ type, attributes: { changed, nid, status, title } }) => (
+                <TableRow key={nid}>
+                  <TableCell>{title}</TableCell>
+                  <TableCell>{this.props.contentTypes[type].name}</TableCell>
+                  <TableCell />
```

Comment:
Should we remove this column for now and create a follow-up issue to add it?

Reply:
This is the user?